backend/requirements.txt

```diff
@@ -23,6 +23,7 @@ django-prometheus==2.2.0
 django-filter==22.1
 pydantic==1.10.1
 redis==4.3.4
+mozilla-django-oidc==3.0.0
```

Comment:
Presumably `3.1.0` will add support for PKCE (it's in master but hasn't been released)